sirepo/server.py

```diff
@@ -584,7 +593,14 @@ def api_updateFolder(self):
             simulation_db.save_simulation_json(r, fixup=False)
         return self.reply_ok()
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec(
+        "require_user",
+        sid="SimId",
+        file_type="FileType",
```

Comment:
Let's keep the arguments in order. as they are defined.

Reply:
24fd189959a5d6096a9482e74f1ec55c9faeb6de